sdk/digitaltwins/ci.yml

```diff
@@ -0,0 +1,30 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - sdk/digitaltwins/
+    exclude:
+      - sdk/digitaltwins/mgmt-v2020_03_01_preview
+
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - sdk/digitaltwins/
+    exclude:
+      - sdk/digitaltwins/mgmt-v2020_03_01_preview
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: digitaltwins
+    Artifacts:
+      - name: azure-digitaltwins
+        groupId: com.azure
```

Comment:
This groupId is different than what is in the POM.xml, not exactly sure how that would change build but they should match.